config/initializers/clearance.rb

```diff
@@ -7,3 +7,26 @@
   config.rotate_csrf_on_sign_in = true
   config.cookie_expiration = ->(_cookies) { 2.weeks.from_now.utc }
 end
+
+class Clearance::Session
+  def current_user
+    return nil unless remember_token.present?
```

Comment:
`config/initializers/clearance.rb:13:16: C: Rails/Blank: Use if remember_token.blank? instead of unless remember_token.present?`